core/src/avm2/globals/flash/desktop/ClipboardFormats.as

```diff
@@ -8,6 +8,18 @@ package flash.desktop
 
     public class ClipboardFormats
```

Comment:
From some testing, nothing in this class is version gated- the class itself, however, is version-gated to SWFv10 and above

Reply:
Thank you, fixed!